Add missing include on binder-add

vkieling opened the pull request:
[utils] add missing include on binder-add

Tomin1 commented:
Thank you for your contribution!

Please reformat the commit message:

- The first line (title) should not have `[utils]` -> `Add missing include on binder-add`
- The second line should have a bug reference which I've created for you in our internal bug tracker -> `[utils] Add missing include on binder-add. JB#57358`

vkieling commented:
> Thank you for your contribution!
> 
> Please reformat the commit message:
> 
> * The first line (title) should not have `[utils]` -> `Add missing include on binder-add`
> * The second line should have a bug reference which I've created for you in our internal bug tracker -> `[utils] Add missing include on binder-add. JB#57358`

I have updated the commit msg and added the bug ID.

Tomin1 reviewed (approved):
LGTM. Thank you!
